examples/appSpec/test/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
```

Comment:
webpack seems kinda overkill if all it is doing is calling babel. This could all be replaced by calling babel directly in the package.json

Reply:
@willemolding let's take the discussion on that point over to : https://github.com/holochain/js-tests-scaffold

Which will be the source of many a test folder configuration

Reply:
I dig it

Reply:
Yeah sure. It's just me on my mission to eliminate unnecessary npm dependencies wherever possible :). We should definitely address it at the source